core/modules/language/language.admin.inc

```diff
@@ -155,20 +155,31 @@ function language_admin_add_form($form, &$form_state) {
 }
 
 /**
- * Editing screen for a particular language.
+ * This function was replaced by language_admin_config_form() but is preserved
+ * for backwards-compatibility.
+ *
+ * @deprecated since 1.13.3
```

Comment:
```suggestion
 * @deprecated since 1.14.0
```

Reply:
I don't think we should be deprecating in a bug-fix release. This would be fine for 1.14.0 though, so if we bump this up I think it should be fine.